core/deployment/src/main/java/io/quarkus/artemis/core/deployment/ArtemisDevServicesBuildTimeConfig.java

```diff
@@ -0,0 +1,71 @@
+package io.quarkus.artemis.core.deployment;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class ArtemisDevServicesBuildTimeConfig {
+
+    /**
+     * If Dev Services for ActiveMQ Artemis has been explicitly enabled or disabled. Dev Services are generally enabled
+     * by default, unless there is an existing configuration present. For Artemis, Dev Services starts a broker unless
```

Comment:
I had to read this sentence a few times before I got the idea
> Dev Services are generally enabled by default, unless there is an existing configuration present.
This is explained in the following sentence? As in, if you are upgrading from a previous version, it won't run. I would make this sentence more future proof.

Reply:
Well, I borrow the sentence from the `KafkaDevServicesBuildTimeConfig` and I think it means that

>The dev servies for ActiveMQ Artemis is enabled by default unless `quarkus.artemis.url` is set or `quarkus.artemis.devservices.enabled` is set to `false` explictly.